### PR TITLE
Restrict user updates to admin users only

### PR DESCRIPTION
**Pull request by fgbruna**

### CODE REVIEW

File: users/models.py [hunk 1]
```diff
@@ -8,5 +8,5 @@ class BaseUser(AbstractUser):
 
 class Resident(BaseUser):
     apartment_number = models.IntegerField()
-    is_approved = models.BooleanField(default=False, disabled=True)
-    is_admin = models.BooleanField(default=False, disabled=True)
+    is_approved = models.BooleanField(default=False)
+    is_admin = models.BooleanField(default=False)
```
Comment (bimartinez) [inline users/models.py:12]: This should be here or in BaseUser?

Comment (gicatalan) [reply users/models.py:12]: 🤔 What do you think @fgbruna ? Probably is_admin in BaseUser and is_approved in Resident but am not sure...

Comment (fgbruna) [reply users/models.py:12]: We should definitely move it to `BaseUser`, but lets keep it there for now

Comment (fgbruna) [reply users/models.py:12]: Frick it, i'm moving `is_admin` to `BaseUser`

**Pull request merged**